examples/README.md

```diff
@@ -0,0 +1,279 @@
+### Tutorial 
+This tutorial will show you how to create, build, install and test a chaincode using Fabric-Private-chaincode repository using the build and test framework.
+
+### Prequisites
+The repository in https://github.com/hyperledger-labs/fabric-private-chaincode has been installed as per [README.md](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/README.md) in FPC-INSTALL-DIR. 
+
+### Create chaincode
+* Create a folder named `helloworld`  in FPC-INSTALL-DIR/examples.
+* Create 2 files, `helloworld_cc.h` and `helloworld_cc.cpp` as follows.  
+
+File:  helloworld_cc.h
+```
+#pragma once
+
+#include <string>
+std::string putData(std::string asset_name, int value, void*ctx);
+std::string marshall (std::string asset_name, uint8_t value);
+std::string getData(std::string asset_name, void*ctx);
+int invoke(const char* args,
+    uint8_t* response,
+    uint32_t max_response_len,
+    uint32_t* actual_response_len,
+    void* ctx);
+
+```
+
+
+File:  helloworld_cc.cpp
```

Comment:
Shouldn't we at least somewhere before providing code say something what we are trying to do (e.g., what is the sample chaincode doing) and at least say something about the programming model, i.e., chaincode has to implement `invoke`, that `shim.h` defines the functionality which the shim provides the chaincode, that the shim is implemented in `libshim.a` and alike?   

RIght now it looks very "black-box" and doesn't really say much more than what you could see by just looking in the examples directory (and, after a bit of searching, in the integration directory for the scripts)? Or put differently, it's not so clear what the didactic value is as-is over a paragraph which says ```lock in examples for sample code and how to build it and in integration on how to run it```?